### PR TITLE
DOCSP-34366 Add docs release notes for w/e November 17

### PR DESCRIPTION
**Pull request by lindseymoore**
### Jira

- https://jira.mongodb.org/browse/DOCSP-34366

### CODE REVIEW

File: docs-release-notes.md [hunk 1]
```diff
@@ -1,3 +1,42 @@
+# November 17, 2023
+
+## Flutter SDK
+
+- Landing Page: Add section for a selection of the SDK's example projects from the Example Projects page.
+- Realm Database/Model Data/Manage Realm Files/Encrypt a Realm: Add note that you cannot encrypt an existing unencrypted realm.
```
Comment (dacharyc) [inline docs-release-notes.md:6]: Nit: this note only mentions a limitation - it doesn't mention that we provide a workaround. I'd probably reword this to focus more on the details we're communicating - not the limitation. Maybe something like:

```suggestion
- Realm Database/Model Data/Manage Realm Files/Encrypt a Realm: Add note that you must encrypt a realm on first open, or use writeCopy to create an encrypted realm. 
```

File: docs-release-notes.md [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+
+## React Native SDK
+
+- Landing Page: Add section for a selection of the SDK's example projects from the Example Projects page.
+- Realm Files/Encrypt a Realm: Add note that you cannot encrypt an existing unencrypted realm. 
+
+## Swift SDK
+
+- Realm Files/Encrypt a Realm: Add note that you cannot encrypt an existing unencrypted realm.
+- CRUD/Threading: Add more detail on using threading with actors.
```
Comment (dacharyc) [inline docs-release-notes.md:31]: This was kind of a snarl of updates, but I'd like to be more explicit here about the change we're recommending for threading + actors.

```suggestion
- CRUD/Threading: Recommend using `ThreadSafeReference` instead of frozen objects with Swift Actors. 
```

File: docs-release-notes.md [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+## React Native SDK
+
+- Landing Page: Add section for a selection of the SDK's example projects from the Example Projects page.
+- Realm Files/Encrypt a Realm: Add note that you cannot encrypt an existing unencrypted realm. 
+
+## Swift SDK
+
+- Realm Files/Encrypt a Realm: Add note that you cannot encrypt an existing unencrypted realm.
+- CRUD/Threading: Add more detail on using threading with actors. 
+- React to Changes: Add details on setting up notifications for changes to different actors.
```
Comment (dacharyc) [inline docs-release-notes.md:32]: This isn't really adding details so much as clarifying/rewording what was already there.
```suggestion
- React to Changes: Clarify details on observing notifications across Swift Actors. 
```

**Review by dacharyc — APPROVED** [review]
I've got a couple of nitpicky wording suggestions, but overall LGTM!

File: docs-release-notes.md (hunk 2 above)
Comment (lindseymoore) [reply docs-release-notes.md:31]: Thank you!
